recipes/bfg/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This doesn't need to be in the recipe directory as `license_file` pulls this content from the source.